capstone/src/main/webapp/script.js

```diff
@@ -1,110 +1,21 @@
 function createMap() {
   const map = new google.maps.Map(
       document.getElementById('map-container'),
-      {center: {lat: 37.7749, lng: -122.4194}, zoom: 13});
-
+      {center: {lat: 37.7749, lng: -122.4194}, zoom: 12});
+  
 	    //**adding zipcode overlay */
         map.data.loadGeoJson('zipcode-data.json');
         map.data.setStyle({visible: false});
+      //**adding precinct overlay */
+        map.data.loadGeoJson('neighborhoods.json');
+        map.data.setStyle({visible: false});
+      {center: {lat: 37.7749, lng: -122.4194}, zoom: 13});
```

Comment:
It seems like you have an unmatched ) here? Not sure what the purpose of this line is

Reply:
(The js also isn't properly formatted and throws console errors, probably due to this issue